index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title id="title">Elon Musk</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+    
+    <body>
+
+        <div id="main">
+            <h1>Elon Musk</h1>
+            <div id="img-div">
+                <img id="image" src="elon.jpeg" alt="Elon musk and friend">
+                <legend>Elon Musk The Dreamer</legend>
+            </div>
+            <div id="tribute-info">
+                <p>
+                    He is the founder, CEO, CTO, and chief designer of SpaceX; early stage investor, CEO, and product architect of Tesla, Inc.; founder of The Boring Company; co-founder of Neuralink; and co-founder and initial co-chairman of OpenAI. A centibillionaire, Musk is one of the richest people in the world.
+                </p>
+
+                <a id="tribute-link" href="#" target="_blank">meet Elon Musk</a>
+            </div>
+        </div>
```

Comment:
It would better if you had used a section tag instead of a div tag for tribute-info. It can help you code clean and give your code a semantic look.